src/main/java/seedu/address/model/summary/Summary.java

```diff
@@ -0,0 +1,102 @@
+package seedu.address.model.summary;
+
+import java.util.HashMap;
+import java.util.List;
+
+import seedu.address.model.AddressBook;
+import seedu.address.model.Model;
+import seedu.address.model.person.CategoryCode;
+import seedu.address.model.person.Person;
+import seedu.address.model.person.Review;
+
+
+/**
+ *  This class contains the logic of retrieving data from different components
+ *  to display as a summary on application start up.
+ */
+public class Summary {
+    private int numberOfContacts;
+    //  private final int percentageRatings;
+    private int percentageReviews;
+    private HashMap<String, Integer> numberCategory;
+
+    private List<Person> personList;
+
+    /**
+     * Constructor to create a summary of the addressBook.
+     * @param addressBook addressBook to summarise.
+     */
+    public Summary(AddressBook addressBook) {
+        this.personList = addressBook.getPersonList();
+        numberOfContacts = setNumberOfContacts();
+        percentageReviews = setPercentageReviews();
+        numberCategory = setNumberCategory();
+    }
+
+    /**
+     * Sets the total number of contacts in the addressBook.
+     * @return size of AddressBook.
+     */
+    private int setNumberOfContacts() {
+        return personList.size();
+    }
+
+    private int setPercentageRatings(Model model) {
```

Comment:
What does this function do? For now it returns a placeholder value of 0, am I right

Reply:
Yes, it returns a 0 for now. I'm awaiting changes done to Rate command/class before implementing this function